src/api/services/auth-service.ts

```diff
@@ -53,22 +53,22 @@ export const authService = {
     return response.data;
   },
 
-  refreshToken: async (refreshToken: string): Promise<Auth.Token> => {
+  refreshToken: async (
+    refreshToken: string
+  ): Promise<Auth.Token & { refresh_token?: string }> => {
     const parameters = new URLSearchParams({
       grant_type: 'refresh_token',
       refresh_token: refreshToken,
     });
     console.log('refreshing token');
-    const response = await authApi.post<Auth.Token>(
-      `/${Endpoints.TOKEN}`,
-      parameters,
-      {
-        auth: {
-          username: CLIENT_ID,
-          password: CLIENT_SECRET,
-        },
-      }
-    );
+    const response = await authApi.post<
+      Auth.Token & { refresh_token?: string }
+    >(`/${Endpoints.TOKEN}`, parameters, {
```

Comment:
просто добавляй / в начало ендпоинтов. если что-то не начинается с / - скорее всего это просто не надо туда добавлять
```suggestion
    >(Endpoints.TOKEN, parameters, {
```